aitelemetry/telemetrywrapper.go

```diff
@@ -207,6 +209,8 @@ func (th *telemetryHandle) TrackLog(report Report) {
 	trace.Tags.User().SetAuthUserId(runtime.GOOS)
 	trace.Tags.Operation().SetId(report.Context)
 	trace.Tags.Operation().SetParentId(th.appVersion)
+	trace.Tags.Application().SetVer(th.appVersion)
```

Comment:
some fields are not exported to kusto. did you check if this visible in kusto? in that case we can remove `trace.Tags.Operation().SetParentId(th.appVersion)`